Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -2311,14 +2283,12 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
         slope.setCustomWidthAndHeight(50, 50);
         platform.setPosition(slope.getX() + slope.getWidth(), slope.getY());
 
-        object.setPosition(0, 0);
+        object.setPosition(0, -5);
```

Comment:
I did a mutant without the separate call to see if the floor following was really due to the follow method. And I think It help me fix an issue, but I also got this test that didn't pass even though it was not related to what it was meant to test. In fact, the 26° platform is 25 pixels high and the object 10x20 pixel, the slope is 0.5 for 1, so the object was in contact with the floor on the 1st frame.
There is already a test for on object on the floor at the beginning. It's on a flat floor though. This could be a valid case but I'm not sure it's worth maintaining it.